### DISCUSSION TITLE
SixLabors.Fonts.FontException: No fonts found installed on the machine.

### DISCUSSION THREAD

**Discussion by WimDeCeuster** [SixLabors/Fonts#436] [General]
And idea on how to resolve / troubleshoot this ?   (v1.0.1)
I

**Comment by tocsoft** [comment]
With very little to go on I would have to guess you have tried to compile your app to run in a docker container and are now surprised that you can't access any system fonts.

If that's the case then that will be your issue, the likelihood that a base docker image ships with any fonts at all is very small as there is zero need for them for 99.9% of their users.

If your application depends on fonts being available then you should always be shipping your own fonts with your application, the SystemFonts collection is really there for prototyping or at the outside desktop scenarios. For any form of server based scenario you should be shipping your own fonts with your app and creating a singleton font collection with your required fonts installed in it.

**Reply by JimBobSquarePants** [reply]
I’d like to note that v1.0.1 is also not supported.
